osd/aws/sts_deleted_provider.json

```diff
@@ -0,0 +1,7 @@
+{
+  "severity": "Error",
+  "service_name": "SREManualAction",
+  "summary": "Action required: Fix AWS Role to maintain SRE support",
+  "description": "Your cluster requires you to take action because the OpenIDConnect provider for this cluster could not be found. Please revert any changes to the OIDC provider to maintain ongoing cluster support. If you require further assistance, please file a support request.",
```

Comment:
We no longer need to include the `If you require further assistance` blurb in each template as it is now automatically included elsewhere in the email. (applicable for both templates)